src/main/resources/avro/MAPRecord.avsc

```diff
@@ -1080,11 +1080,10 @@
         "string"
       ]
     },
-  {
-    "name":
-    "messages",
-    "type": {
-      "name": "MessagesIngestMessageArray",
+    {
+      "name": "messages",
+      "type": {
+        "name": "MessagesIngestMessageArray",
```

Comment:
This just fixes some spacing for legibility.

Reply:
YOU MONSTER